misc/docker-ci/check.mk

```diff
@@ -1,4 +1,4 @@
-CONTAINER_NAME=h2oserver/h2o-ci:ubuntu1604
+CONTAINER_NAME?=h2oserver/h2o-ci:ubuntu1604
```

Comment:
We would no longer need this once we adopt the change above.

Reply:
Yes, it is no longer need it. Reverted: 4f143b2